featuretools/entityset/deserialize.py

```diff
@@ -90,12 +32,14 @@ def description_to_entityset(description, **kwargs):
     entityset = EntitySet(description['id'])
 
     last_time_index = []
-    for entity in description['entities'].values():
-        entity['loading_info']['params'].update(kwargs)
-        # If path is None, an empty dataframe will be created for entity.
-        description_to_entity(entity, entityset, path=path)
-        if entity['properties']['last_time_index']:
-            last_time_index.append(entity['id'])
+    for df in description['dataframes'].values():
+        if path is not None:
+            data_path = os.path.join(path, 'data', df['name'])
+            dataframe = read_woodwork_table(data_path, validate=False, **kwargs)
+        else:
+            dataframe = empty_dataframe(df)
```

Comment:
I don't think this matters because there's no data to actually load here, but I just wanted to point out that the woodwork table description as stored in `data_description.json` that gets used for the empty dataframe case has none of the loading info (location, type, encoding, etc) that is in the `woodwork_typing_info.json` as stored in the `data` directory.